integration-tests/load/vrfv2plus/gun.go

```diff
@@ -50,7 +52,8 @@ func (m *SingleHashGun) Call(_ *wasp.Generator) *wasp.Response {
 	}
 
 	//randomly increase/decrease randomness request count per TX
-	randomnessRequestCountPerRequest := deviateValue(*m.testConfig.GetVRFv2PlusConfig().General.RandomnessRequestCountPerRequest, *m.testConfig.GetVRFv2PlusConfig().General.RandomnessRequestCountPerRequestDeviation)
+	configCopy := deepcopy.MustAnything(m.testConfig).(types.VRFv2PlusTestConfig)
```

Comment:
why we need to do copy here? also, dont think it should be copied here

Reply:
Just to avoid having any side effect on other tests

Reply:
I see, do you suggest directly modifying m.testConfig

Reply:
Fixed this. Avoided copying the config

Reply:
Avoided the copy here